standalone/pherry/src/lib.rs

```diff
@@ -437,22 +438,35 @@ pub async fn get_authority_with_proof_at(
 ) -> Result<AuthoritySetChange> {
     // Storage
     let id_key = phaxt::dynamic::storage_key("Grandpa", "CurrentSetId");
-    // Authority set
-    let value = api
-        .rpc()
-        .storage(GRANDPA_AUTHORITIES_KEY, Some(hash))
-        .await?
-        .expect("No authority key found")
-        .0;
+    let alt_authorities_key = phaxt::dynamic::storage_key("Grandpa", "Authorities");
+    let old_authorities_key: &[u8] = b":grandpa_authorities";
+
+    static OLD_AUTH_KEY_PRIOR: AtomicBool = AtomicBool::new(true);
```

Comment:
```suggestion
    // Try the old grandpa storage key first if true. Otherwise try the new key first.
    static OLD_AUTH_KEY_PRIOR: AtomicBool = AtomicBool::new(true);
```